<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>182</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>182</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Game Mode:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>ModeSelector</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QComboBox" name="AIColorSelector">
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>AI Color :</string>
         </property>
         <property name="buddy">
          <cstring>AIColorSelector</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>First Player :</string>
         </property>
         <property name="buddy">
          <cstring>PvAIFirstSelector</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="PvAIFirstSelector">
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="PvAI_PlayerName">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Player Name :</string>
         </property>
         <property name="buddy">
          <cstring>PvAI_PlayerName</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" alignment="Qt::AlignTop">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>First Player:</string>
         </property>
         <property name="buddy">
          <cstring>PvPFirstSelector</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignTop">
        <widget class="QComboBox" name="PvPFirstSelector">
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Black Name :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Red Name :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="PvP_RedPlayer">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="PvP_BlackPlayer">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="ModeSelector">
     <item>
      <property name="text">
       <string>PvAI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PvP</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="NewGame_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="NewGame_Play">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ModeSelector</tabstop>
  <tabstop>AIColorSelector</tabstop>
  <tabstop>PvAIFirstSelector</tabstop>
  <tabstop>PvAI_PlayerName</tabstop>
  <tabstop>NewGame_Play</tabstop>
  <tabstop>NewGame_Cancel</tabstop>
  <tabstop>PvPFirstSelector</tabstop>
  <tabstop>PvP_RedPlayer</tabstop>
  <tabstop>PvP_BlackPlayer</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>NewGame_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ModeSelector</sender>
   <signal>activated(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
